css-anchor-position-1/Overview.bs

```diff
@@ -667,37 +658,16 @@ This means a positioned element
 will <em>not</em> be aligned with its anchor
 if any of the scrollers are <em>not</em> at their initial positions.
 
-The 'anchor-scroll' property allows an author to compensate for this,
-without losing the performance benefits of the separate scrolling thread,
+To compensate for this without losing
+the performance benefits of the separate scrolling thread,
 so long as the positioned element
-is only anchoring to a single anchor element.
-Its values are:
-
-<dl dfn-type=value dfn-for=anchor-scroll>
-	: <dfn>none</dfn>
-	:: No effect.
-
-	: <dfn>default</dfn>
-	:: Behaves identically to <<anchor-element>>,
-		but draws its value from 'anchor-default' on the element.
-
-	: <dfn><<anchor-element>></dfn>
-	::
-		Selects a [=target anchor element=]
-		the same as ''anchor()'',
-		which will be compensated for in positioning and fallback.
-</dl>
-
-Note: When the element uses 'anchor-default'
-or has an [=implicit anchor element=],
-authors can often avoid explicitly setting an 'anchor-scroll' value
-because the initial value is ''anchor-scroll/default''.
+is only anchoring to elements in the same [=scroll container=],
```

Comment:
Hm, this is a new restriction. Do you mean something like "as long as all the [=target anchor elements=] of [=anchor functions=] used on the element have the same nearest ancestor [=scroll container=]"?

Reply:
I didn't mean to add a new restriction, but just trying to reword the existing statement and make it more precise: the compensation works well only if all anchors are in the same scroller.

The existing statement says it works well only if there's only one anchor, which isn't entirely correct.

Reply:
Ah, then it shouldn't be stated here in the list of restrictions, but rather in a Note.

Reply:
Actually, I don't see a very straightforward condition for this compensation to work well. Even if there's only one anchor, it can be e.g.

```css
top: calc(anchor(bottom) * 0.9)
```

which will appear detached with the anchor during scrolling.

I'll just remove this entirely.